ModuleDescriptor.json

```diff
@@ -26,6 +26,10 @@
           "methods": [ "GET" ],
           "pathPattern": "/eholdings/jsonapi/titles*"
         },
+        {
+          "methods": [ "GET" ],
+          "pathPattern": "/eholdings/jsonapi/customer-resources*"
```

Comment:
if the resource type is `customerResources` maybe we should use camelCase here too, as much as it pains me.